script/install/debootstrap.bash

```diff
@@ -171,5 +171,48 @@ echo "$chroot_cmd update-alternatives --install /usr/bin/g++ g++ /usr/bin/g++-9
 chroot "$ISOLATE_ROOT" update-alternatives --install /usr/bin/g++ g++ /usr/bin/g++-9 75
 # gcc 9 done
 
-# let user know that chroot installs are finished
+[ -z "$TRAVIS" ] && bash script/confirm.bash 'Install JavaScript V8 (submissions in V8 will fail without this!)' && {
+  # JavaScript V8 engine compile and install
+  echo "Preparing to compile and install the JavaScript V8 engine."
+  echo
+  echo "If this fails, visit https://v8.dev/docs/build and follow the instructions."
+  echo "This may take a while ..."
+  echo
+  set -x
+
+  : Making temporary directory
+  mkdir ~/v8-build
+  cd ~/v8-build
+
+  : Installing depot_tools
+  git clone https://chromium.googlesource.com/chromium/tools/depot_tools.git
+  export PATH="$PATH:$PWD/depot_tools"
+  gclient
+
+  : Getting V8 source code
+  mkdir v8
+  cd v8
+  fetch v8
+  cd v8
+
+  : Installing additional build dependencies
+  gclient sync
+  ./build/install-build-deps.sh
+
+  : Compiling V8
+  ./tools/dev/v8gen.py x64.release -- v8_use_external_startup_data=false
+  ninja -C out.gn/x64.release d8
+
+  : Copying executable into place
+  cp ./out.gn/x64.release/d8 "$ISOLATE_ROOT"/usr/bin/d8
+
+  : Cleaning up
+  cd ~
+  rm -r v8-build
```

Comment:
```suggestion
  rm -rf v8-build
```
For me it needed `-f` to remove read-only files. Did it work for you without `-f`?

Reply:
It did work without `-f` for me but I wouldn't mind adding it.

Reply:
From private discussion it turns to be my fault, I was running the commands directly instead of using `install.bash` (which runs `debootstrap.bash` as root, and then `rm` doesn't prompt about read-only files). I still suggest `-f` in case we want to run these commands as non-root.